app/scripts/views/base.js

```diff
@@ -100,6 +100,7 @@ function (Cocktail, _, Backbone, $, p, AuthErrors,
       this.relier = options.relier;
       this.broker = options.broker;
       this.user = options.user;
+      this.notifications = options.notifications;
```

Comment:
If notifications are only used in Views with the AvatarMixin, you can also override AvatarMixin's initialize function to capture the reference to `notifications`. Both AvatarMixin's and the mixed in View's initialize [will be called](https://github.com/onsi/cocktail#how-are-colliding-functions-called).